storage/tests/perf/README.md

```diff
@@ -0,0 +1,21 @@
+# storage benchwrapp
+
+main.py is a gRPC wrapper around the storage library for benchmarking purposes.
+
+## Running
+
+```bash
+$ export STORAGE_EMULATOR_HOST=localhost:8080
```

Comment:
Could you add http:// to this address? I just realized today that it's required.

Reply:
c30db8f62689c276c2885a3dccd7d7eeed53944a